dpnp/backend/custom_kernels_random.cpp

```diff
@@ -28,6 +28,8 @@
 #include "backend_utils.hpp"
 #include "queue_sycl.hpp"
 
+#include <vector>
```

Comment:
I commented it above. It looks like it is good idea to do not use STL in API.
It is acceptable inside "C kernels" but not in API